requirements-dev.txt

```diff
@@ -4,3 +4,5 @@ pylint>=2.6.0
 pytest>=6.0.0
 pytest-cov>=2.10.1
 click==8.0.2
+mypy>=1.3.0
```

Comment:
How did you decide which requirements to put here vs requirements-mypy.txt? Seems like they should all be in dev at this point

Reply:
reqs.txt -> project deps
reqs-dev.txt -> deps for development machines
reqs-mypy.txt -> stubs required by mypy 

I can combine mypy/dev if the -mypy convention isn't desired

Reply:
I'm agnostic on whether they're combined, but should mypy itself be in the mypy section?

Reply:
Combined -mypy.txt into -dev.txt